subsys/bluetooth/host/hci_core.c

```diff
@@ -2455,7 +2445,7 @@ static void process_events(struct k_poll_event *ev, int count)
 
 static void hci_tx_thread(void *p1, void *p2, void *p3)
 {
-	static struct k_poll_event events[EV_COUNT] = {
+	struct k_poll_event events[EV_COUNT] = {
```

Comment:
Not sure by how much the `hci_tx_thread`'s program stack size needs increase, if retaining the `static` does not fault, lets keep it back.

Reply:
Seems to work fine with the `static`, so I'm bringing it back.